cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractConfiguration.java

```diff
@@ -20,123 +20,130 @@
  * Abstract class for processing and generating configuration passed by the user.
  */
 public abstract class AbstractConfiguration {
-    private static final Logger log = LogManager.getLogger(AbstractConfiguration.class.getName());
+    private static final ReconciliationLogger RECONCILIATION_LOGGER = ReconciliationLogger.create(AbstractConfiguration.class.getName());
```

Comment:
Do we need to call it `RECONCILIATION_LOGGER` instead of just `LOGGER`? We do not seem to have any other logger here so the long name seems unnecessary.

Reply:
I was wondering about abbreviating to `RLOGGER`.